auraed/src/runtime/cells/cell.rs

```diff
@@ -41,25 +72,28 @@ impl Cell {
         Self { name, cgroup, executables: Default::default() }
     }
 
-    pub fn free(self) -> Result<()> {
-        self.cgroup.delete()?;
+    pub fn free(self) -> Result<(), CellError> {
```

Comment:
`type Result<T> = Result<T, CellError>` can simplify some of the code here.. at least it makes it easy to spot when we're being inconsistent (if we are).

Reply:
Will add it